observation_portal/accounts/forms.py

```diff
@@ -32,11 +33,7 @@ def save(self, commit=True):
             user=new_user_instance,
             title=self.cleaned_data['title'],
             institution=self.cleaned_data['institution'],
-            education_user=self.cleaned_data['education_user'],
```

Comment:
Is the intention to manually update who is an education user from now on?

Reply:
The `education_user` flag is not used for any of our projects. It could be completely removed. I think this was a misunderstanding from the early Valhalla spec.

Reply:
Oh interesting I did not know that. If we don't need it, we should probably remove that field from the Profile model entirely. I'll want to check with a couple of others before I do that, just in case. Not that doing that is necessary for this PR, just a TODO.